tests/unit/test_exceptions.py

```diff
@@ -54,6 +55,18 @@ def test_message(self):
         )
 
 
+class TestInvalidFlairTemplateID:
+    def test_inheritance(self):
+        assert isinstance(InvalidFlairTemplateID(None), ClientException)
+
+    def test_message(self):
+        assert (
+            str(InvalidFlairTemplateID("123"))
+            == "The flair template id ``123`` is invalid. If you are "
+            "trying to create a flair, please use the ``add`` method."
+        )
+
+
```

Comment:
Not a big deal, but these tests don't seem to add anything in my view. They test something that is plainly defined elsewhere in code and they won't fail until that code is changed. If that code is changed, it's probably deliberate and so the test failure would not be helpful.

Reply:
I don't think we need to test inheritance, but it would be nice to test the message.

Reply:
The reason being is that I could assume the message is different, and not realize I changed something.